prime-dotnet-webapi/Configuration/PrimeConfigurationObjects.cs

```diff
@@ -108,4 +108,9 @@ public class SentryConfiguration
     {
         public string Dsn { get; set; }
     }
+
+    public class ProviderTeamConfiguration
```

Comment:
I think this (and similar) should be "provider enrolment team"; less ambiguous, as precise as we know it

Reply:
Updated as suggested.